job/validators.py

```diff
@@ -357,7 +364,7 @@ def cross_clean_pv(pvmodel):
             if self.models["Settings"].off_grid_flag==False:
                 self.models["ElectricLoad"].critical_load_pct = 0.5
             else:
-                self.models["ElectricLoad"].critical_load_pct = 1.0
+                self.models["ElectricLoad"].critical_load_pct = 1.0 # TODO: this always needs to be true for off-grid
```

Comment:
@rathod-b another instance of needing to override user inputs. Critical_load_pct must always = 1 when off_grid_flag is true. See the validation we do in REopt.jl [here](https://github.com/NREL/REopt.jl/blob/8d7beb8a916d1cdf8571c268d44a6ef86dc07a7b/src/core/electric_load.jl#L125).